### PR TITLE
Add new target FLYWOOF405NANO,F405PRO Config

### PR DESCRIPTION
**Pull request by flywoo**
Add new target FLYWOOF405NANO,F405PRO Config

### CODE REVIEW

**Review by haslinghuis — DISMISSED** [review]

File: configs/default/FLWO-FLYWOOF405PRO.config [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+set beeper_od = OFF
+set max7456_spi_bus = 3
+set dashboard_i2c_bus = 1
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW180FLIP
+set gyro_1_align_pitch = 1800
+set gyro_1_align_yaw = 1800
+set gyro_2_spibus = 1
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF405PRO.config:118]: Where's the second gyro?

File: configs/default/FLWO-FLYWOOF405NANO.config [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+set ibata_scale = 170
+set beeper_inversion = ON
+set beeper_od = OFF
+set max7456_spi_bus = 3
+set dashboard_i2c_bus = 1
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = DEFAULT						   					   					   
+set gyro_2_spibus = 1
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF405NANO.config:116]: Where's the second gyro?

File: configs/default/FLWO-FLYWOOF405NANO.config [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+set battery_meter = ADC
+set ibata_scale = 170
+set beeper_inversion = ON
+set beeper_od = OFF
+set max7456_spi_bus = 3
+set dashboard_i2c_bus = 1
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = DEFAULT
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF405NANO.config:115]: Why youe keep default here?

**Review by DusKing1 — CHANGES_REQUESTED** [review]

File: configs/default/FLWO-FLYWOOF405PRO.config (hunk 1 above)
Comment (flywoo) [reply configs/default/FLWO-FLYWOOF405PRO.config:118]: I will fix it, thanks

File: configs/default/FLWO-FLYWOOF405NANO.config (hunk 2 above)
Comment (flywoo) [reply configs/default/FLWO-FLYWOOF405NANO.config:116]: I will fix it, thanks

File: configs/default/FLWO-FLYWOOF405NANO.config (hunk 3 above)
Comment (flywoo) [reply configs/default/FLWO-FLYWOOF405NANO.config:115]: I will fix it, thanks

File: configs/default/FLWO-FLYWOOF405PRO.config [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+set beeper_inversion = ON
+set beeper_od = OFF
+set max7456_spi_bus = 3
+set dashboard_i2c_bus = 1
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW180FLIP
+set gyro_1_align_pitch = 1800
+set gyro_1_align_yaw = 1800
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF405PRO.config:117]: EOL

File: configs/default/FLWO-FLYWOOF405NANO.config [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 170
+set beeper_inversion = ON
+set beeper_od = OFF
+set max7456_spi_bus = 3
+set dashboard_i2c_bus = 1
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
```
Comment (DusKing1) [inline configs/default/FLWO-FLYWOOF405NANO.config:114]: EOL

**Review by DusKing1 — CHANGES_REQUESTED** [review]

File: configs/default/FLWO-FLYWOOF405NANO.config (hunk 5 above)
Comment (flywoo) [reply configs/default/FLWO-FLYWOOF405NANO.config]: fixed, thanks

File: configs/default/FLWO-FLYWOOF405PRO.config (hunk 4 above)
Comment (flywoo) [reply configs/default/FLWO-FLYWOOF405PRO.config]: fixed, thanks

**Review by haslinghuis — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**